From 924b7830007c1aee51142b393a46f234749e931d Mon Sep 17 00:00:00 2001
From: Juan Pablo Miceli <micelijuanpablo@gmail.com>
Date: Fri, 11 Oct 2024 00:33:12 +0000
Subject: [PATCH] Update rules_fuzzing

---
 MODULE.bazel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MODULE.bazel b/MODULE.bazel
index a2843f809..8dee5a311 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -26,5 +26,5 @@ bazel_dep(name = "googletest", version = "1.13.0", repo_name = "com_google_googl
 bazel_dep(name = "protobuf", version = "26.0", repo_name = "com_google_protobuf")
 bazel_dep(name = "re2", version = "2024-02-01", repo_name = "com_googlesource_code_re2")
 bazel_dep(name = "rules_cc", version = "0.0.9")
-bazel_dep(name = "rules_fuzzing", version = "0.5.1")
+bazel_dep(name = "rules_fuzzing", version = "0.5.2")
 bazel_dep(name = "rules_proto", version = "4.0.0")
