From 12368d8c2512138b95741d39605790b56e98ee32 Mon Sep 17 00:00:00 2001
From: Trashcan <trashcan69666@gmail.com>
Date: Fri, 8 Nov 2024 14:54:29 -0500
Subject: [PATCH] fix

---
 tcmalloc/internal/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tcmalloc/internal/BUILD b/tcmalloc/internal/BUILD
index 3a6c019..398d9a2 100644
--- a/tcmalloc/internal/BUILD
+++ b/tcmalloc/internal/BUILD
@@ -14,7 +14,7 @@
 #
 # Internal libraries used for the implementation and testing of TCMalloc.

-load("@rules_cc//cc:defs.bzl", "cc_proto_library")
+load("@com_google_protobuf//bazel:cc_proto_library.bzl", "cc_proto_library")
 load("@rules_proto//proto:defs.bzl", "proto_library")
 load("//tcmalloc:copts.bzl", "TCMALLOC_DEFAULT_COPTS")
 load("//tcmalloc:variants.bzl", "create_tcmalloc_benchmark")
--
2.47.0

